src/scripts/devhub.zig

```diff
@@ -150,6 +172,7 @@ fn devhub_metrics(shell: *Shell, cli_args: CLIArgs) !void {
                 .value = checksum_message_size_max_us,
                 .unit = "us",
             },
+            .{ .name = "ci pipeline duration", .value = ci_pipeline_duration_s.?, .unit = "s" },
```

Comment:
Let's swap it with the build one, the build was always ment to be a proxy for CI. I _think_ the order of graph will be determined by order in this slice.

Reply:
By swap, I assume you meant keep both and swap the order - let me know if otherwise. I think it might be interesting to track debug build times too, but I'll look at that in a follow up.

Reply:
Yeah, keep both, swap the order!